src/obu.c

```diff
@@ -413,7 +413,10 @@ avifBool avifSequenceHeaderParse(avifSequenceHeader * header, const avifROData *
         avifBitsInit(&bits, obus.data, obus.size);
 
         // obu_header()
-        avifBitsRead(&bits, 1); // obu_forbidden_bit
+        const uint8_t obu_forbidden_bit = avifBitsRead(&bits, 1);
```

Comment:
Nit: The return type of `avifBitsRead()` is `uint32_t`, so it is simpler to use the `uint32_t` type here (to avoid a type conversion).

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/2146.